cmlreaders/readers/readers.py

```diff
@@ -149,19 +152,54 @@ class MontageReader(BaseCMLReader):
     @staticmethod
     def _flatten_row(data: dict, labels: List[str], i: int) -> pd.DataFrame:
         entry = data[labels[i]].copy()
-        atlases = entry.pop('atlases')
-        atlas_row = json_normalize(atlases)
-        atlas_row.index = [i]
-        row = pd.concat([pd.DataFrame(entry, index=[i]), atlas_row], axis=1)
-        return row
+        try:
+            atlases = entry.pop('atlases')
+            atlas_row = json_normalize(atlases)
+            atlas_row.index = [i]
+            row = pd.concat([pd.DataFrame(entry, index=[i]), atlas_row], axis=1)
+            return row
+        except KeyError:
+            # Ramulator pairs.json files are more minimal and do not include
+            # the atlas locations
+            return pd.DataFrame(entry, index=[i])
 
     def as_dataframe(self):
         from concurrent.futures import ProcessPoolExecutor as Pool
         from functools import partial
         from multiprocessing import cpu_count
 
-        with open(self._file_path) as f:
-            data = json.load(f)[self.subject][self.data_type]
+        if self.data_type == "pairs":
+            try:
+                rerefable = is_rerefable(self.subject, self.experiment,
+                                         self.session, self.localization,
+                                         self.montage, self.rootdir)
+            except FileNotFoundError:
+                # We get this error when testing with file_path specified, so
+                # don't worry about it here.
+                import warnings
+                warnings.warn(
+                    "Couldn't determine if EEG data can be rereferenced. "
+                    "This can happen if you didn't specify an experiment and/or"
+                    " session, or if you're passing file_path explicitly. "
+                    "Proceeding with the assumption that rereferencing is "
+                    "possible.", UserWarning
```

Comment:
you should double check that this will actually get printed when FileNotFoundError is caught. I believe I ran into an issue with UserWarning not being something that results in an actual message being displayed, but I could be thinking of something else